WalletWasabi/Blockchain/Keys/HdPubKey.cs

```diff
@@ -14,8 +14,10 @@ namespace WalletWasabi.Blockchain.Keys
 	[JsonObject(MemberSerialization.OptIn)]
 	public class HdPubKey : NotifyPropertyChangedBase, IEquatable<HdPubKey>
 	{
+		public const int DefaultHighAnonymitySet = int.MaxValue;
```

Comment:
There must be a default value to signal the anonymity set was not yet set for this.  

One way to go about it is make `int` nullable and so the default value would be `null`. That'd work fine.
Another way to go about it is to leave the default value for `int`, which is `0`. That'd work fine also because zero would be an invalid anonset.

Why I decided to not go with the conventions and rather use `int.MaxValue` is an intuition that when the user generates an address, then I assume that the anonymity set of that address is infinite as nobody else knows about that address at that point so display layer code doesn't need to handle anything but for example in the UI it would be accurate even if the default is forgotten to be handled.

Finally another way to go about it is to make anonymity set its own class/struct.

I don't feel strongly about one way or another, IMO it's fine like this, doesn't really matter.